.changeset/ten-maps-dance.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/azure-client": minor
```

Comment:
Would this be a major change?

Reply:
I believe @tylerbutler 's guidance was to just leave as minor since we don't use `changeset` to version our packages.